samples/python/tutorial_code/imgProc/hough_line_transform/hough_line_transform.py

```diff
@@ -6,8 +6,8 @@
 edges = cv.Canny(gray,50,150,apertureSize = 3)
 
 lines = cv.HoughLines(edges,1,np.pi/180,200)
-for line in lines:
-    rho,theta = line[0]
+for line in lines[0]:
```

Comment:
This code draws one line only.

Please take a look on existed code from Python samples: https://github.com/opencv/opencv/blob/3.4.6/samples/python/houghlines.py

Probably this code should be dropped and tutorial should use Python sample directly.